test/xds/xds_server_serving_mode_test.go

```diff
@@ -384,6 +359,22 @@ func (s) TestServerSideXDS_ServingModeChanges(t *testing.T) {
 	waitForSuccessfulRPC(ctx, t, cc2)
 }
 
+func createStubServer(t *testing.T, lis net.Listener, creds credentials.TransportCredentials, modeChangeOpt grpc.ServerOption, bootstrapContents []byte) *stubserver.StubServer {
```

Comment:
looks like you are using this only for one test. Can't it be used for other tests as well?

Reply:
Used for other tests.